en/advanced/advanced_position_ctrl_tuning.md

```diff
@@ -0,0 +1,101 @@
+<!-- Fixes page load bug in notation, courtesy https://github.com/GitbookIO/plugin-mathjax/issues/34#issuecomment-349453673 -->
+<script>
+gitbook.events.bind("page.change", function() {
+    MathJax.Hub.Queue(["Typeset",MathJax.Hub]);
+}
+</script>
+
+# Advanced Position Control Tuning
+
+Before doing any non-pid related control tuning, please follow first the instruction [PID-tuning](https://docs.px4.io/en/advanced_config/pid_tuning_guide_multicopter.html) to find gains for the attitude and position control loops.
+It is not recommended to use the advanced position control tuning parameters as a fix for bad tracking, position control related vibration etc. The PID-Gains should be chosen such that tracking is as tight as possible. The 
+tuning discussed in this tutorial only effects the position/velocity setpoint to improve user experience for **MANUAL** and **AUTO** related position control flight modes. To turn off any higher level tuning, please set the following parameters accordingly and continue with [PID-tuning](https://docs.px4.io/en/advanced_config/pid_tuning_guide_multicopter.html):
+
```

Comment:
So basically you're saying "do PID tuning first. If you want to do PID tuning first set these params to the  values shown?

IMO the whole of an overview should not be about not using the document - it should be about what the doc is for. If can include warnings, but if these are extensive they should be in another section - possibly referenced by a note from the intro.

What I would do is:
1. move the "Short background" up as overview. 
2. To that add a short note referencing the material currently in this overview
   > **Note** [PID tuning] must be performed before using this document - see [preconditions].
3. As next section, have "## Preconditions" explaining all the stuff currently in the overview.

Reply:
but short background is also not really an overview of the document, because it is about the controller loops and not about advance tuning. 
I will move both.